### PR TITLE
fixes: translation of open assignment string using i18n

### LINKED ISSUE #2509: Translation of "Open assignment" using i18n
https://github.com/CircuitVerse/CircuitVerse/blob/59bf87ccb3f9cc19443e72af9084077c6334566f/app/views/lti/open_incv.html.erb#L17

### PR DESCRIPTION
**Pull request by vedant-jain03**
Fixes #2509 

#### Describe the changes you have made in this PR -

- added open_assignment key in en.yml
- added translation function in open_incv.html.erb

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by vedant-jain03** [comment]
@pavanjoshi914 @ayan-biswas0412 Kindly Review

**Review by gr455 — APPROVED** [review]
Looks Good

**Comment by pavanjoshi914** [comment]
Add translations in hi.yml as well if you know Hindi

**Comment by vedant-jain03** [comment]
> Add translations in hi.yml as well if you know Hindi

Thier is not hi.yml file for lti , can I make one and add this ?

**Comment by vedant-jain03** [comment]
> Add translations in hi.yml as well if you know Hindi

Sure

**Comment by vedant-jain03** [comment]
@pavanjoshi914 Kindly review

**Review by gr455 — APPROVED** [review]

**Review by pavanjoshi914 — CHANGES_REQUESTED** [review]

File: config/locales/views/lti/hi.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+en:
```
[the remark this answers is not in the document]
Comment (vedant-jain03) [reply config/locales/views/lti/hi.yml]: @pavanjoshi914 Done

**Comment by pavanjoshi914** [comment]
Good work @vedant-jain03. please add screenshot of output to this PR

@tachyons  LGTM

**Review by pavanjoshi914 — APPROVED** [review]

**Comment by vedant-jain03** [comment]
Screenshot (149)

**Review by Kartikay304 — APPROVED** [review]

**Review by tachyons — APPROVED** [review]

**Pull request merged**